website/server/middlewares/errorHandler.js

```diff
@@ -16,7 +16,7 @@ module.exports = function errorHandler (err, req, res, next) { // eslint-disable
   // Otherwise try to identify the type of error (mongoose validation, mongodb unique, ...)
   // If we can't identify it, respond with a generic 500 error
   let responseErr = err instanceof CustomError ? err : null;
-
+  console.log(err)
```

Comment:
I assume this is unintentional.

Reply:
Yep. Lint caught it too. I'll remove it during my next updates.